crates/rooch-framework-tests/tests/cases/entry_function/entry_function_with_return_value.move

```diff
@@ -0,0 +1,54 @@
+//# init --addresses creator=0x42
+
+//# publish
+module creator::test {
+
+    entry public fun test_entry_function_return_value_u8():u8{
```

Comment:
I think the entry function should not return values. @WGB5445  @steelgeek091

Reply:
Yes, the verifier also checks inside if the entry function has a return value, an error will be thrown. So here, the error message should be determined in the exp file.

https://github.com/rooch-network/rooch/blob/925ca13c486a6b81c8f866d2762ff5b55727ab1b/moveos/moveos-verifier/src/verifier.rs#L153

Reply:
But there is no error reported in the actual exp file.
It should be a problem when publishing
I think this needs to be fixed in another pr